Apps/frontend/cypress/e2e/2-control-panel/preset-config.cy.js

```diff
@@ -6,7 +6,7 @@ describe("everything visible on front page", () => {
   });
 
   it("displays btn", () => {
-    cy.get("#managePresets").should("be.visible");
+    cy.get("#settingsButton").should("be.visible");
```

Comment:
I'd much prefer to test with `data-cy` instead of `id`.

Reply:
Done